rust/agents/relayer/Cargo.toml

```diff
@@ -9,15 +9,15 @@ coingecko = { git = "https://github.com/hyperlane-xyz/coingecko-rs", tag = "2022
 config = "~0.13.3"
 serde = {version = "1.0", features = ["derive"]}
 serde_json = { version = "1.0", default-features = false }
-ethers = { git = "https://github.com/hyperlane-xyz/ethers-rs", tag = "2023-01-10-04" }
```

Comment:
We need to merge the other PR and create a tag that gets used in all the cargo.toml files before this PR get's merged. Creating this comment to prevent automerge by accident.

Reply:
Should we merge upstream changes too?

Reply:
As I said in the other PR, I don't think so because at this time they have not made another release and we have not found any cases where we need one of their features in main that is not in the current release.